src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER =
+             DateTimeFormatter.ofPattern("dd.MM.yyyy", Locale.ENGLISH);
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate startDate = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate endDate = LocalDate.parse(dateTo, FORMATTER);
+        StringBuilder report = new StringBuilder();
+
+        report.append("Report for period ").append(dateFrom).append(" - ").append(dateTo);
```

Comment:
Unnecessary separation between `StringBuilder` initialization and `append()` calls.

hint: you might want to break this method chain into multiple lines, new line on each method call, to increase readiness:
```java
StringBuilder reportBuilder = new StringBuilder()
    .append("some text")
    .append("another text");
```